### PR TITLE
Meeting 2022 11 03

### PR DESCRIPTION
**Pull request by maltaesousa**

### CODE REVIEW

File: content/pages/98_meetings_de.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Status: hidden
 
 Hier befindet sich die Liste der Meetings der Gruppe. In jeder Seite befinden sich die Entscheidungen / Bemerkungen des Meeting.
 
+### 2022
+
+* [Yverdon-les-Bains, 24.11.2022](meeting_minutes/2022-11-03)
```
Comment (rbovard) [inline content/pages/98_meetings_de.md]: Date issue?

File: content/pages/98_meetings_fr.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Status: hidden
 
 Ici se trouve une liste des meetings du groupe. Chaque page regroupe les décisions / remarques prises.
 
+### 2022
+
+* [Yverdon-les-Bains, 24.11.2022](meeting_minutes/2022-11-03)
```
Comment (rbovard) [inline content/pages/98_meetings_fr.md]: Date issue?

**Review by rbovard — CHANGES_REQUESTED** [review]
Looks good, thanks a lot.

Just a date typo to fix ;)

**Comment by maltaesousa** [comment]
@rbovard thanks for the review!

**Pull request merged**